huggingface-gpt2/README.md

```diff
@@ -0,0 +1,156 @@
+# Accelerate GPT2 fine-tuning with ONNX Runtime
+
+This example uses ONNX Runtime to fine-tune the GPT2 PyTorch model maintained at https://github.com/huggingface/transformers.
+
+You can run the training in Azure Machine Learning or locally.
+
+## Setup
+
+1. Clone this repo
+
+    ```bash
+    git clone https://github.com/microsoft/onnxruntime-training-examples.git
+    cd onnxruntime-training-examples/huggingface-gpt2
+    ```
+
+2. Clone download code and model
+
+    ```bash
+    git clone https://github.com/huggingface/transformers.git
+    cd transformers/
+    git checkout 9a0a8c1c6f4f2f0c80ff07d36713a3ada785eec5
+    ```
+
+3. Update with ORT changes
+
+    ```bash
+    git apply ../ort_addon/src_changes.patch
+    cp -r ../ort_addon/ort_supplement/* ./
+    ```
+
+## Download and prepare data
+
+The following are a minimal set of instructions to download one of the datasets used for GPT2 finetuning for the language modeling task.
+
+Download the word-level dataset [WikiText-103](https://blog.einstein.ai/the-wikitext-long-term-dependency-language-modeling-dataset/) for this sample.
+Refer to the readme at [transformers](https://github.com/huggingface/transformers/tree/master/examples/language-modeling#language-model-training) for additional details.
+
+Download the data and export path as $DATA_DIR: 
+```bash
+    export DATA_DIR=/path/to/downloaded/data/
+```
```

Comment:
is there a `wget` required to download the data?

Reply:
The huggingface repo doesn't explicitly mention the wget, so we don't as well. The users can choose how they download it, we mention the source of the dataset and link to the original huggingface readme for more details. This will avoid changes when the links to the dataset change.